examples/build_image.sh

```diff
@@ -26,6 +26,8 @@ federated-learning-mistnet-yolo-aggregator.Dockerfile
 federated-learning-mistnet-yolo-client.Dockerfile
 federated-learning-surface-defect-detection-aggregation.Dockerfile
```

Comment:
In this commit `federated-learning-surface-defect-detection-aggregation.Dockerfile`/`federated-learning-surface-defect-detection-train.Dockerfile` have been renamed, so we need to remove this line from `build_image.sh`